From 504c9db6657c5dc1634637bf38cb4132fd909fc7 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Tue, 17 Jun 2025 09:11:48 +0300
Subject: [PATCH] [SW-228312] g2: update files to match sival repo

Sival repo update gaudi2 GAUDI2_PROTBITS_DATA112
this patch updates zephyr to match the update
values

Change-Id: I465538fdf1ca7981332dc34c145d8be341c7ade5
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/drivers/misc/hl_security/gaudi2/gaudi2_protbits_configs.h b/drivers/misc/hl_security/gaudi2/gaudi2_protbits_configs.h
index 2ded869..2c4164c 100644
--- a/drivers/misc/hl_security/gaudi2/gaudi2_protbits_configs.h
+++ b/drivers/misc/hl_security/gaudi2/gaudi2_protbits_configs.h
@@ -462,7 +462,7 @@
 		0x100001ff, 0x60000 }
 
 #define GAUDI2_PROTBITS_DATA112 { \
-		 0xff00000f, 0x10ffff, 0x8000, 0x60000 }
+		 0xff00000f, 0xff10ffff, 0x80ff, 0x60000 }
 
 #define GAUDI2_PROTBITS_DATA113 { \
 		 0xffffff, 0xffff0000, 0x3fd50000, 0xffffff, 0xffff0000, 0x3fd50000, \
